config/software/readline.rb

```diff
@@ -0,0 +1,15 @@
+name "readline"
```

Comment:
why do we need this?

I think in the python2 software def we remove the libreadline shared object or something similar, can you have a look?

Reply:
Indeed, I will try that for python3 as well

Reply:
It worked 👍